test/models/scholarship_test.rb

```diff
@@ -0,0 +1,16 @@
+require 'test_helper'
+
+class ScholarshipTest < ActiveSupport::TestCase
+  test 'unexpired returns unexpired scholarships' do
+    nil_date = create :scholarship, close_time: nil
+    active_date = create :scholarship, close_time: 3.days.from_now
+
+    assert_equal Scholarship.unexpired, [nil_date, active_date]
+  end
+
+  test 'unexpired doesn\'t return expired scholarships' do
+    expired_date = create :scholarship, close_time: 3.days.ago
+
+    assert_not_equal Scholarship.unexpired, [expired_date]
```

Comment:
Assertions about "not equal" can come back to bite. An assertion that the results do not include the expired scholarship is a bit more robust.

```ruby
assert_not_includes Scholarship.unexpired, expired_date
```